packages/flutter_tools/lib/src/device.dart

```diff
@@ -1145,6 +1131,7 @@ class DebuggingOptions {
   final bool ipv6;
   final String? google3WorkspaceRoot;
   final bool printDtd;
+  final WebDevServerConfig? devConfig;
```

Comment:
`devConfig` is a bit of a generic name. I'd prefix this with `web` everywhere to clarify this is for web:

```suggestion
  final WebDevServerConfig? webDevConfig;
```